src/CMakeLists.txt

```diff
@@ -14,6 +14,13 @@ project(parse_fasm)
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../third_party/antlr4/runtime/Cpp/cmake)
```

Comment:
@litghost -- I need some help here dealing with these compiler flags.

I would like to;
 - Add `-Wimplicit-fallthrough=0` if it is supported on non-Windows
 - Add `-Wno-attributes` if it is supported on non-Windows
 - Add `/std:c++latest` on Windows..

(IE I would like to get rid of the `add_flags` function in the setup.py Python file above and just have cmake correctly understand the flags.)

Reply:
https://docs.microsoft.com/en-us/cpp/build/reference/compiler-options?view=msvc-160

Reply:
https://en.cppreference.com/w/cpp/language/attributes/fallthrough

If ANTLR could emit `[[fallthrough]]`, `-Wimplicit-fallthrough=0` is not required.

Reply:
PIC support might be possible via: https://cmake.org/cmake/help/v3.0/prop_tgt/POSITION_INDEPENDENT_CODE.html

Reply:
CMake has explicit C++ standard select support: https://cmake.org/cmake/help/latest/prop_tgt/CXX_STANDARD.html

Reply:
I'm going to defer this to #51